src/pages/index.js

```diff
@@ -34,69 +38,151 @@ const modalImageElement = imagePreviewModal.querySelector(".modal__image-preview
 const modalCaption = imagePreviewModal.querySelector(".modal__image-caption");
 const imagePreviewCloseButton = imagePreviewModal.querySelector(".modal__close");
 
+// Confirm Delete Modal
+const trashIcon = document.querySelector("#confirm-delete-modal");
+const trashSubmitButton = trashIcon.querySelector("#confirm-delete-submit")
 
+// Avatar Edit Modal
+const avatarEditModal = document.querySelector("#avatar-edit-modal");
 
 const cardTemplate = document.querySelector("#card-template").content.firstElementChild;
 
 
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f34fc940-6444-4c84-9643-915e876889fb",
+    "Content-Type": "application/json",
+  }
+}); 
+
+let cardSection;
+
+api
+  .getInitialCards()
+  .then((res) => {
+    cardSection = new Section({
+      items: res,
+      renderer: (cardData) => {
+        const cardEl = renderCard(cardData);
+        cardSection.addItem(cardEl);
+      },
+    },  
+  ".cards__list"
+  );
+  
+  cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err); 
+  });
+  
+const profileUserInfo = new UserInfo(".profile__title", ".profile__description", ".profile__image");    
+  
+api
+  .loadUserInfo()
+  .then((res) => {
+    profileUserInfo.getUserInfo();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
-// Create instances of the classes
-const cardSection = new Section({
-  items: initialCards,
-  renderer: (cardData) => {
-    const cardEl = renderCard(cardData);
-    cardSection.addItem(cardEl);
-  },
-},  
-".cards__list"
-);
 
-cardSection.renderItems();
 
-const profileEditFormPopup = new PopupWithForm("#profile-edit-modal", handleProfileEditSubmit);
-profileEditFormPopup.setEventListeners();
 
-const editFormElement = profileEditModal.querySelector(".modal__form_edit");
-const editFormValidator = new FormValidator(validationSettings, editFormElement);
-editFormValidator.enableValidation();
+const editFormElement = profileEditModal.querySelector(".modal__form_edit");  

```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms